test/passing/tests/cinaps.ml

```diff
@@ -18,10 +18,12 @@ let x = 1
 (*$*)
 let y = 2
 
-(*$ ;; #use "import.cinaps"
+(*$
+  #use "import.cinaps";;
 
-  ;; List.iter all_fields ~f:(fun (name, type_) -> printf "\nexternal get_%s
-  : unit -> %s = \"get_%s\"" name type_ name) *)
+  List.iter all_fields ~f:(fun (name, type_) ->
+      printf "\nexternal get_%s\n  : unit -> %s = \"get_%s\"" name type_ name )
+*)
```

Comment:
This works because of the smarter parsing ? I think we should add a `.ref` file to keep the original.

Reply:
done